### PR TITLE
[Doc] Add Chinese doc for `scannet_det.md`

### PR DESCRIPTION
**Pull request by THU17cyz**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs_zh-CN/datasets/scannet_det.md [hunk 1]
```diff
@@ -1 +1,304 @@
 # 3D 目标检测 Scannet 数据集
+
+## 数据集准备
+
+请参考 ScanNet 的[指南](https://github.com/open-mmlab/mmdetection3d/blob/master/data/scannet/README.md/) 以查看总体流程。
```
Comment (Tai-Wang) [inline docs_zh-CN/datasets/scannet_det.md:5]: “以查看总体流程”前面的空格可以删一下

File: docs_zh-CN/datasets/scannet_det.md [hunk 2]
```diff
@@ -1 +1,304 @@
...
+        aligned_bboxes = extract_bbox(aligned_mesh_vertices, object_id_to_segs,
+                                      object_id_to_label_id, instance_ids)
+    ...
+
+    return mesh_vertices, label_ids, instance_ids, unaligned_bboxes, \
+        aligned_bboxes, object_id_to_label_id, axis_align_matrix
+
+```
+
+在从每个场景的扫描文件提取数据后，如果原始点云点数过多，可以将其下采样（比如到 50000 个点），但对于三维语义分割任务点云不会被下采样。此外，在 `nyu40id` 标准之外的不合法语义标签或者可选的 `DONOT CARE` 类别标签应被过滤。最终，点云文件、语义标签、实例标签和真实物体的集合应被存储于 `.npy` 文件中。
```
Comment (Tai-Wang) [inline docs_zh-CN/datasets/scannet_det.md:128]: 对于三维语义分割任务点云不会被下采样 -> 在三维语义分割任务中点云不会被下采样

File: docs_zh-CN/datasets/scannet_det.md [hunk 3]
```diff
@@ -1 +1,304 @@
...
+    dict(
+        type='Collect3D',
+        keys=[
+            'points', 'gt_bboxes_3d', 'gt_labels_3d', 'pts_semantic_mask',
+            'pts_instance_mask'
+        ])
+]
+```
+
+- `GlobalAlignment`：输入的点云在施加了坐标轴平行的矩阵后应被转换为坐标轴平行的。
```
Comment (Tai-Wang) [inline docs_zh-CN/datasets/scannet_det.md:293]: 应被转换为与坐标轴平行的形式

File: docs_zh-CN/datasets/scannet_det.md [hunk 4]
```diff
@@ -1 +1,304 @@
...
+    ...
+
+    return mesh_vertices, label_ids, instance_ids, unaligned_bboxes, \
+        aligned_bboxes, object_id_to_label_id, axis_align_matrix
+
+```
+
+在从每个场景的扫描文件提取数据后，如果原始点云点数过多，可以将其下采样（比如到 50000 个点），但在三维语义分割任务中，点云不会被下采样。此外，在 `nyu40id` 标准之外的不合法语义标签或者可选的 `DONOT CARE` 类别标签应被过滤。最终，点云文件、语义标签、实例标签和真实物体的集合应被存储于 `.npy` 文件中。
+
+### 提取 ScanNet RGB 色彩数据
```
Comment (Wuziyi616) [inline docs_zh-CN/datasets/scannet_det.md:130]: 我觉得是不是标注成 `提取 ScanNet RGB 色彩数据 (可选的)` 会比仅在下面内容注明可选来得清晰一些？

File: docs_zh-CN/datasets/scannet_det.md [hunk 5]
```diff
@@ -1 +1,304 @@
...
+│   ├── scenexxxx_xx
+│   │   ├── xxxxxx.txt
+│   │   ├── xxxxxx.jpg
+│   │   ├── intrinsic.txt
+├── scannet_infos_train.pkl
+├── scannet_infos_val.pkl
+├── scannet_infos_test.pkl
+```
+
+- `points/xxxxx.bin`：下采样后，未与坐标轴平行（即没有对齐）的点云。因为 ScanNet 3D 检测任务将与坐标轴平行的点云作为输入，而 ScanNet 3D 语义分割任务将对齐前的点云作为输入，我们选择存储对齐前的点云和它们的对齐矩阵。请注意：在 3D 检测的预处理流程 `GlobalAlignment` 后，点云就都是与坐标轴平行的了。
```
Comment (Wuziyi616) [inline docs_zh-CN/datasets/scannet_det.md:225]: 对 `GlobalAlignment` 加一个到 `transforms_3d.py` 里对应代码的链接可能会更清晰一些？

File: docs_zh-CN/datasets/scannet_det.md [hunk 6]
```diff
@@ -1 +1,304 @@
...
+        - annotations['unaligned_dimensions']：depth 坐标系下与坐标轴不平行的三维包围框的大小。
+        - annotations['unaligned_gt_boxes_upright_depth']：depth 坐标系下与坐标轴不平行的三维包围框。
+        - annotations['index']：所有真实物体的索引，范围为 [0, K)。
+        - annotations['class']：所有真实物体类别的标号，范围为 [0, 18)，形状为 [K, ]。
+- `scannet_infos_val.pkl`：验证集上的数据信息，与 `scannet_infos_train.pkl` 格式完全一致。
+- `scannet_infos_test.pkl`：测试集上的数据信息，与 `scannet_infos_train.pkl` 格式几乎完全一致，除了缺少标注。
+
+## Training pipeline
+
+ScanNet 上 3D 物体检测的经典流程如下：
```
Comment (Wuziyi616) [inline docs_zh-CN/datasets/scannet_det.md:250]: `经典流程`-->`典型流程`？

File: docs_zh-CN/datasets/scannet_det.md (hunk 4 above)
Comment (Wuziyi616) [reply docs_zh-CN/datasets/scannet_det.md:130]: 英文文档里经常见到 `### xxx (Optional)` 的感觉

File: docs_zh-CN/datasets/scannet_det.md (hunk 5 above)
Comment (Wuziyi616) [reply docs_zh-CN/datasets/scannet_det.md:225]: 请也在英文文档加上链接

File: docs_zh-CN/datasets/scannet_det.md [hunk 7]
```diff
@@ -1 +1,304 @@
...
+        ])
+]
+```
+
+- `GlobalAlignment`：输入的点云在施加了坐标轴平行的矩阵后应被转换为与坐标轴平行的形式。
+- `PointSegClassMapping`：训练中，只有合法的类别 ID 才会被映射到类别标签，比如 [0, 18)。
+- 数据增强:
+    - `IndoorPointSample`：下采样输入点云。
+    - `RandomFlip3D`：随机左右或前后翻转点云。
+    - `GlobalRotScaleTrans`: 旋转输入点云，对于 ScanNet 角度通常落入 [-5, 5] （度）的范围；并放缩输入点云，对于 ScanNet 比例通常为 1.0；最后平移输入点云，对于 ScanNet 通常位移量为 0。
```
Comment (Wuziyi616) [inline docs_zh-CN/datasets/scannet_det.md:298]: 多加一点注释会不会清晰一些？
`对于 ScanNet 比例通常为 1.0 **(即不做缩放)**`
`对于 ScanNet 通常位移量为 0 **(即不做位移)**`

File: docs_zh-CN/datasets/scannet_det.md [hunk 8]
```diff
@@ -1 +1,304 @@
...
+
+### 提取 ScanNet RGB 色彩数据
+
+通过提取ScanNet RGB 色彩数据，对于每个场景我们加载 RGB 图像与配套 4x4 位姿矩阵、单个 4x4 相机内参矩阵的集合。请注意，这一步是可选的，除非要运行多视图物体检测，否则可以略去这步。
+
+```shell
+python extract_posed_images.py
+```
+
+1201 个训练样本，312 个验证样本和 100 个测试样本中的每一个都包含一个单独的 `.sens` 文件。比如说，对于场景 `0001_01` 我们有 `data/scannet/scans/scene0001_01/0001_01.sens`。对于这个场景所有图像和位姿数据都被提取至 `data/scannet/posed_images/scene0001_01`。具体来说，该文件夹下会有 300 个 xxxxx.jpg 格式的图像数据，300 个 xxxxx.txt 格式的相机位姿数据和一个单独的 `intrinsic.txt` 内参文件。通常来说，一个场景包含数千张图像。默认情况下，我们只会提取其中的 300 张，从而只占用少于 100 Gb 的空间。要想提取更多图像，请使用 `--max-images-per-scene` 参数。
```
[the remark this answers is not in the document]
Comment (THU17cyz) [reply docs_zh-CN/datasets/scannet_det.md:138]: 这个问题也问问 @Wuziyi616 @Tai-Wang ？

[the remark this answers is not in the document]
Comment (Tai-Wang) [reply docs_zh-CN/datasets/scannet_det.md:138]: 通常是翻译成位姿的

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**